File: src/TileType.cpp
#include <SFML/Graphics/Rect.hpp>

#include <TileType.hpp>

TileType::TileType(sf::IntRect _texture_rect, sf::IntRect _ghost_texture_rect) {
	texture_rect = _texture_rect;
	ghost_texture_rect = _ghost_texture_rect;
}

File: src/structs/tile_type.rs
use sfml::graphics::IntRect;

pub struct TileType {
    pub texture_rect: IntRect,
    pub ghost_texture_rect: IntRect,
}

impl TileType {
    pub fn new(texture_rect: IntRect, ghost_texture_rect: IntRect) -> Self {
        Self {
            texture_rect,
            ghost_texture_rect,
        }
    }
}
